keps/sig-node/1287-in-place-update-pod-resources/kep.yaml

```diff
@@ -23,12 +23,26 @@ approvers:
   - "@mwielgus"
 editor: TBD
 creation-date: 2018-11-06
-last-updated: 2020-01-14
+last-updated: 2021-02-05
 status: implementable
 see-also:
   - "/keps/sig-node/2273-kubelet-container-resources-cri-api-changes"
 replaces:
 superseded-by:
 
+# PRR
+prr-approvers:
+  - "@ehashman"
+feature-gates:
+  - name: InPlacePodVerticalScaling
+    components:
+      - kube-apiserver
+      - kube-scheduler
+      - kubelet
+disable-supported: true
+milestone:
+  alpha: "v1.22"
+  beta: "v1.23"
+  stable: "v1.24"
```

Comment:
FWIW it will take longer than 1.24 to graduate this, due to the feature flag on the kubelet (as we support n-2 version skew between kubelets and API servers), but this is a nit.

Reply:
Changed it to v1.25